resultbrowser/src/js/resultbrowser.js

```diff
@@ -95,211 +183,233 @@
-            $element.addClass("current");
+        if (!element.classList.contains("current")) {
+            actionlist.querySelectorAll(":scope .current").forEach((el) => el.classList.remove("current"))
+            element.classList.add("current");
         }
     }
 
     function expandCollapseAction(element) {
         // lazily create the requests
-        if ($('ul.requests', element).length == 0) {
+        if (!element.querySelector("ul.requests")) {
             createRequestsForAction(element);
         }
 
         // show/hide the requests
-        $('ul.requests', element).slideToggle(200, resizeContent);
+        toggle(element.querySelector("ul.requests"));
 
         // show/hide the requests
-        $('.expander', element).toggleClass("expanded");
+        element.querySelector(".expander").classList.toggle("expanded");
     }
 
     function createRequestsForAction(actionElement) {
         // build requests element
-        var requests       = $('<ul class="requests"></ul>'),
-            $actionElement = $(actionElement),
-            action         = $actionElement.data('json'),
-            actionRequests = action && action.requests || [];
+        let requests = document.createElement("ul");
+        requests.classList.add("requests");
+        let action = dataStore.fetchData(actionElement);
+        let actionRequests = action && action.requests || [];
 
-        $actionElement.append(requests);
+        actionElement.appendChild(requests)
 
         // make sure, we do not see it building up
-        requests.hide();
+        hide(requests);
 
         // ok, we have to add the data from the json object to it
-        for ( var i = 0, l = actionRequests.length; i < l; i++) {
-            var request = actionRequests[i];
-            var name = request.name;
-            var contentTypeClass = determineContentTypeClass(request.mimeType, request.responseCode);
-            var protocolClass = determineProtocolClass(request.url);
-            var title = "[" + request.responseCode + "] " + request.url;
+        for (const request of actionRequests) {
+            let name = request.name;
+            let contentTypeClass = determineContentTypeClass(request.mimeType, request.responseCode);
+            let protocolClass = determineProtocolClass(request.url);
+            let title = "[" + request.responseCode + "] " + request.url;
 
-            var requestElement = $('<li class="request" title="' + htmlEncode(title) + '"><span class="name ' + htmlEncode(contentTypeClass)
-                                    + ' ' + htmlEncode(request.requestMethod) + ' ' + htmlEncode(protocolClass) +'">' + htmlEncode(name) + '</span></li>');
+            let requestElement = document.createElement("li");
+            requestElement.classList.add("request");
+            requestElement.title = htmlEncode(title);
+
+            let nameElement = document.createElement("span");
```

Comment:
```suggestion
            const nameElement = document.createElement("span");
```

Reply:
Feel free to review the other `let` statements as well. As long as these variables are not reassigned a value, they should be declared w/ `const`.

Reply:
Done